File: fontes/backend/node/retaguarda-sh/src/alexa/alexa-slot.ts
import { AlexaSlotObjeto } from "./alexa-slot-objeto";

export class AlexaSlot {

	nomeUsuario: AlexaSlotObjeto;

	/**
	* Constructor
	*/
	constructor(objetoJson: object) {
		if (objetoJson != null) {
			this.nomeUsuario = objetoJson['nomeUsuario'];
		}	
	}

}

File: fontes/backend/java/retaguarda_sh/src/main/java/com/t2ti/retaguarda_sh/model/alexa/AlexaSlot.java
package com.t2ti.retaguarda_sh.model.alexa;

import java.io.Serializable;

public class AlexaSlot implements Serializable {
	private static final long serialVersionUID = 1L;
	
	private AlexaSlotObjeto nomeUsuario;

	public AlexaSlot() {}

	public AlexaSlotObjeto getNomeUsuario() {
		return nomeUsuario;
	}

	public void setNomeUsuario(AlexaSlotObjeto nomeUsuario) {
		this.nomeUsuario = nomeUsuario;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}


}